core/tee/tee_svc_storage.c

```diff
@@ -177,7 +177,7 @@ static TEE_Result tee_svc_storage_create_file(struct tee_ta_session *sess,
 	int tmp;
 	uint32_t cflags = TEE_FS_O_WRONLY | TEE_FS_O_CREATE;
 
-	if (flags & TEE_DATA_FLAG_EXCLUSIVE)
+	if (!(flags & TEE_DATA_FLAG_OVERWRITE))
```

Comment:
I don't really get this, EXCLUSIVE and OVERWRITE has the exact same value, but still you need to negate the expression now. Since nothing else has been changed in this function this makes me a little bit suspicious.

If think what you try to fulfill here is the section in GP spec saying:
```
o If the flag is absent and the object exists, then the function SHALL return
TEE_ERROR_ACCESS_CONFLICT .
```
If yes, where do we return TEE_ERROR_ACCESS_CONFLICT? Could you please clarify what I'm missing?